streaming/java/BUILD.bazel

```diff
@@ -147,6 +147,7 @@ define_java_module(
         "@maven//:com_google_protobuf_protobuf_java",
         "@maven//:commons_io_commons_io",
         "@maven//:de_ruedigermoeller_fst",
+	"@maven//:net_java_dev_jna_jna",
```

Comment:
what does this used for?

Reply:
> what does this used for?

import jna in streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/util/JniUtils.java